src/viam/logging.py

```diff
@@ -1,11 +1,43 @@
+import asyncio
 import logging
 import sys
 from copy import copy
-from logging import DEBUG, ERROR, FATAL, INFO, WARN, WARNING  # noqa: F401
+from datetime import datetime
+from logging import DEBUG, ERROR, FATAL, INFO, WARN, WARNING, LogRecord  # noqa: F401
 from typing import Dict
 
+import viam
+
 LOG_LEVEL = INFO
 LOGGERS: Dict[str, logging.Logger] = {}
+_MODULE_PARENT = None
```

Comment:
(minor) should we type-annotate this? It takes a bit of digging around to really understand what `_MODULE_PARENT` actually _is_.

Reply:
update: I see this was discussed below! If @njooma's suggestion works here then would love to see annotation; if not, no worries!